open_in_mpv/open_in_mpv.py

```diff
@@ -1,10 +1,9 @@
 # SPDX-License-Identifier: MIT
 from functools import lru_cache
-from os.path import dirname, exists, expanduser, isdir, join as path_join
-from typing import Any, Callable, Mapping, TextIO
+from os.path import dirname, exists, expanduser, expandvars, isdir, join as path_join
+from typing import Any, Final, BinaryIO, Callable, cast, Mapping, TextIO
```

Comment:
`cast` should come last (uppercase imports go first). This is the last thing to change.

Reply:
not a problem, pylint didn't complain for this.

Reply:
Run `yarn qa` to see all of the issues.